arm-sql/2014-04-01/swagger/sql.core.json

```diff
@@ -1330,6 +1330,9 @@
             "description": "The name of the database for which setting the transparent data encryption applies."
           },
           {
+            "$ref": "#/parameters/TransparentDataEncryptionConfigNameParameter"
```

Comment:
A parameter reference like this one is missing for the "get" operation below at https://github.com/Azure/azure-rest-api-specs/pull/1005/files#diff-79c4f982670d02607125a6b94696be04R1379
See https://travis-ci.org/Azure/azure-rest-api-specs/jobs/209193853#L973